File: laprak1/php/main.php
<?php
require('Mahasiswa.php');
$Hlist = array();

$temp = new Mahasiswa();

$temp->set_Nama("ardyn");
$temp->set_Kelamin("L");
$temp->set_Nim(201);
$temp->set_Fakultas("FPMIPA");
$temp->set_Prodi("Ilkom");

array_push($Hlist, $temp);

$temp2 = new Mahasiswa();
$temp2->set_Nama("yusuf");
$temp2->set_Kelamin("L");
$temp2->set_Nim(202);
$temp2->set_Fakultas("FPMIPA");
$temp2->set_Prodi("Ilkom");

array_push($Hlist, $temp2);

$temp3 = new Mahasiswa();
$temp3->set_Nama("faiz");
$temp3->set_Kelamin("L");
$temp3->set_Nim(203);
$temp3->set_Fakultas("FPMIPA");
$temp3->set_Prodi("Ilkom");

array_push($Hlist, $temp3);

$temp4 = new Mahasiswa();
$temp4->set_Nama("ayes");
$temp4->set_Kelamin("L");
$temp4->set_Nim(204);
$temp4->set_Fakultas("FPMIPA");
$temp4->set_Prodi("Ilkom");

array_push($Hlist, $temp4);

echo '<table border = 1>
		<thead>
			<tr>
				<th>Nama</th>
				<th>Jenis Kelamin</th>
				<th>NIM</th>
				<th>Fakultas</th>
				<th>Prodi</th>
			</tr>
		</thead>
		<tbody>';

foreach ($Hlist as $mahasiswa) {
    echo '<tr>
			<td>' . $mahasiswa->get_Nama() . '</td>
			<td>' . $mahasiswa->get_Kelamin() . '</td>
			<td>' . $mahasiswa->get_Nim() . '</td>
			<td>' . $mahasiswa->get_Fakultas() . '</td>
			<td>' . $mahasiswa->get_Prodi() . '</td>
		</tr>';
}
echo '
	</tbody>
</table>';

// echo "Berikut isi dari list Mahasiswa: ";
// echo "<hr>";
// foreach ($Hlist as $mahasiswa) {
//     echo $mahasiswa->get_Nama() . " " . $mahasiswa->get_Nim() . " " .  $mahasiswa->get_Kelamin() . " " . $mahasiswa->get_Fakultas() . " " . $mahasiswa->get_Prodi() . "<hr>";
// }

File: laprak1/java/main.java
import java.util.*;

public class main {

    public static void main(String[] args) {

        List<Mahasiswa> mlist = new ArrayList<>();

        Scanner scanner = new Scanner(System.in);

        System.out.print("Masukkan berapa banyak orang mahasiswa : ");
        int n = scanner.nextInt();

        for (int i = 0; i < n; i++) {
            Mahasiswa temp = new Mahasiswa();

            String name = scanner.next();
            int nim = scanner.nextInt();
            String fakultas = scanner.next();
            String prodi = scanner.next();

            temp.setNama(name);
            temp.setNim(nim);
            temp.setFakultas(fakultas);
            temp.setProdi(prodi);

            mlist.add(temp);
        }

        System.out.println("Berikut isi dari list (awal): ");
        for (Mahasiswa mahasiswa : mlist) {
            System.out.println(mahasiswa.getNama() + " " + mahasiswa.getNim() + " " +
                    mahasiswa.getFakultas() + " " + mahasiswa.getProdi());
        }
        System.out.println();

        System.out.println("Jika anda mau mengedit data dalam list");
        System.out.println("Dapat menggunakan beberapa masukkan :");
        System.out.println("TAMBAH   --> untuk menambah isi list mahasiswa");
        System.out.println("KURANG   --> untuk menghapus isi list mahasiswa");
        System.out.println("RUBAH    --> untuk mengubah isi list mahasiswa");
        System.out.println();

        int c;
        System.out.println("Masukkan input (1) jika ingin melakukkan perubahan dan (2) jika ingin berhenti");
        c = scanner.nextInt();

        String instruksi;
        while (c == 1) {
            Mahasiswa order = new Mahasiswa();
            System.out.print("\nMasukkan instruksi: ");
            instruksi = scanner.next();
            if (instruksi.equals("TAMBAH")) {
                System.out.println("Masukkan nilai (nim) (nama) (fakultas) (prodi)");
                int nim = scanner.nextInt();
                String name = scanner.next();
                String fakultas = scanner.next();
                String prodi = scanner.next();
                order.setNama(name);
                order.setNim(nim);
                order.setFakultas(fakultas);
                order.setProdi(prodi);
                order.tambah(mlist, order);
            } else if (instruksi.equals("KURANG")) {
                System.out.println("Masukkan nilai (nim)");
                int nim = scanner.nextInt();
                order.setNim(nim);
                order.kurang(mlist, order);
            } else if (instruksi.equals("RUBAH")) {
                System.out.println("Masukkan nilai (nim)*yg sudah ada, lalu (nama) (fakultas) (prodi)*baru");
                int nim = scanner.nextInt();
                String name = scanner.next();
                String fakultas = scanner.next();
                String prodi = scanner.next();
                order.setNama(name);
                order.setNim(nim);
                order.setFakultas(fakultas);
                order.setProdi(prodi);
                order.ubah(mlist, order);
            }
            System.out.println("jika ingin lanjut (1) jika ingin stop (2)");
            System.out.print("masukkan : ");
            c = scanner.nextInt();
        }

        System.out.println("\nBerikut isi dari list (Akhir): ");
        for (Mahasiswa mahasiswa : mlist) {
            System.out.println(mahasiswa.getNama() + " " + mahasiswa.getNim() + " " +
                    mahasiswa.getFakultas() + " " + mahasiswa.getProdi());
        }
    }
}
